tests/boot/boot_from_pxe.pm

```diff
@@ -109,7 +109,7 @@ sub run {
     save_screenshot;
 
     if ((check_var('BACKEND', 'ipmi') && !get_var('AUTOYAST')) || get_var('SES5_DEPLOY')) {
-        my $ssh_vnc_wait_time = get_var('SES5_DEPLOY') ? 300 : 210;
+        my $ssh_vnc_wait_time = 300 ;
```

Comment:
Please, remove space after 300, it broke tidy.

Reply:
okay, thanks!